python/tests/conftest.py

```diff
@@ -14,6 +14,14 @@ def source_root():
     raise RuntimeError('Cannot find the source folder')
 
 
+@pytest.fixture()
+def use_tmpdir(tmpdir):
```

Comment:
This functionality was just merged in _libres_, please use this on instead: https://github.com/equinor/libres/blob/eb8a93b0d6b2b770720d5c273b2647b781f6c655/python/tests/utils/__init__.py#L16